iml-system-test-utils/src/iml.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2020 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use crate::{try_command_n_times, CheckedStatus};
+use std::io;
+use tokio::{fs, process::Command};
+
+const IML_DOCKER_PATH: &str = "/etc/iml-docker";
```

Comment:
Just FYI: duplicates https://github.com/whamcloud/integrated-manager-for-lustre/pull/1712/files#diff-58bc3ec70ec0819d14c433040c8e99e7R10

Reply:
You're right, i'll make it public so it's not duplicated.